nucypher/cli/characters/alice.py

```diff
@@ -1,18 +1,22 @@
+from itertools import chain
+
 import click
 from constant_sorrow.constants import NO_BLOCKCHAIN_CONNECTION
 
-from nucypher.blockchain.eth.interfaces import BlockchainInterfaceFactory
 from nucypher.characters.banners import ALICE_BANNER
+from nucypher.characters.control.specifications import AliceSpecification, CharacterSpecification
 from nucypher.cli import actions, painting, types
-from nucypher.cli.actions import get_nucypher_password, select_client_account, get_client_password
+from nucypher.cli.actions import get_nucypher_password, select_client_account, get_client_password, echo_schema
 from nucypher.cli.config import nucypher_click_config
 from nucypher.cli.types import NETWORK_PORT, EXISTING_READABLE_FILE, EIP55_CHECKSUM_ADDRESS
 from nucypher.config.characters import AliceConfiguration
 from nucypher.config.keyring import NucypherKeyring
 
 
+
 @click.command()
 @click.argument('action')
+@click.option('--options', help="Export JSON type schema", is_flag=True)
```

Comment:
The name of this option and the help don't really help me understand either here.